docs/docs/health-checks.md

```diff
@@ -60,6 +60,8 @@ Mesos-level health checks (`MESOS_HTTP`, `MESOS_HTTPS`, `MESOS_TCP`, and `COMMAN
 
 - Mesos-level health checks require tasks to listen on the container's loopback interface in addition to whatever interface they require. If you run a service in production, you will want to make sure that the users can reach it.
 
+- Marathon currently does NOT support the combination of Mesos and Marathon level health checks.
```

Comment:
Do you know if we reject such combinations?

Reply:
https://jira.mesosphere.com/browse/MARATHON-8369

good catch.. this JIRA is to "reject" this combination.   Currently it is not rejected and it doesn't work properly if used.